bots/image-download

```diff
@@ -253,8 +252,9 @@ def wait_lock(target):
         raise TimeoutError("timed out waiting for concurrent downloads of %s\n" % target)
 
 def download_images(image_list, force, quiet, state, store):
-    if not os.path.exists(DATA):
-        os.makedirs(DATA)
+    data_dir = testvm.get_images_data_dir()
+    if not os.path.exists(data_dir):
+        os.makedirs(data_dir)
```

Comment:
I've run into similar TOCTOU races recently; so while you are at  it, could you please use `exist_ok=True` instead? We can rely on py3 now.